st2client/st2client/models/core.py

```diff
@@ -333,7 +333,7 @@ def re_run(self, execution_id, parameters=None, **kwargs):
 
 class TriggerInstanceResourceManager(ResourceManager):
     @add_auth_token_to_kwargs_from_env
-    def re_emit(self, trigger_instance_id):
+    def re_emit(self, trigger_instance_id, **kwargs):
```

Comment:
I did what we are doing for re_run in executions. Scroll this file and you'll see. I hate this kwargs abuse but whatever.